recipes/magnum-plugins/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class TestPackage(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["IMPORTER_PLUGINS_FOLDER"] = os.path.join(self.deps_user_info["magnum-plugins"].plugins_basepath, "importers").replace("\\", "/")
+        # STL file taken from https://www.thingiverse.com/thing:2798332
```

Comment:
```
License
Conan C/C++ Package Manager 3D logo bydanimtbis licensed under theBSD Licenselicense.
```

Reply:
Where should I mention this license? Shall I add a `README.md` inside `test_package` mentioning the model author and license?

Reply:
nowhere, I believe, link is already there